WISE-Community/WISE-Client
# feat(Peer Grouping Authoring): Implement authoring popup

You may want to test this along with the WISE-API branch ```issue-114-rename-peer-group-activity```.

You will need to change these project content field names in your project.json.
- In the project content change peerGroupActivities to peerGroupings
- In the component content change peerGroupActivityTag to peerGroupingTag

## Changes
- Implemented Peer Grouping authoring
- Renamed PeerGroupActivity to PeerGroupings

## Test
- Make Peer Chat and Show Group Work authoring can set the Peer Grouping Tag
- Make sure you can create, edit, delete Peer Groupings
- Make sure you can author the Peer Grouping Tag for all components in their advanced authoring popup
- Make sure Peer Chat in the VLE works like before

Closes #552
